Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -1125,5 +1125,6 @@
     <string name="test_api_body_other_keys">"Other key"</string>
     <!-- NOTR -->
     <string name="test_api_calculate_risk_level">"Calculate Risk Level"</string>
+    <string name="test_api_country_filter_status">Country filter applied for: \n%1$s</string>
```

Comment:
Why are we adding english strings to the localized files here?
Wouldn't just adding it to the default be enough as we have no translations for this?
Is this related to the translation setup?

Reply:
Maybe I think is it better if we remove this completely from the translation setup

Reply:
@harambasicluka Thoughts? Should test related strings be hardcoded to not popup in the translation tooling?

Reply:
I've removed the translation in the test fragment, because as I checked how the other labels are done, I was the only one added it to the translation, now it the same as the other labels are done.